ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -117,6 +117,8 @@ type ssh_daemon_type = {
     "options" ? ssh_daemon_options_type
     "comment_options" ? ssh_daemon_options_type
     "sshd_path" ? string
+    # if true and sshd doesn't exist, will skip config validation
```

Comment:
The opposite of always is never but "never != (not always)".  Or to put it another way: always = false does not imply never.